hudi-hadoop-common/src/main/java/org/apache/hudi/common/util/ParquetUtils.java

```diff
@@ -447,7 +447,7 @@ private static Comparable<?> convertToNativeJavaType(PrimitiveType primitiveType
       synchronized (primitiveType.stringifier()) {
         // Date logical type is implemented as a signed INT32
         // REF: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
-        return java.sql.Date.valueOf(
+        return java.time.LocalDate.parse(
```

Comment:
`LocalDate` should be have with `java.sql.Date.valueOf()`, so why do we need this change?
https://docs.oracle.com/javase/8/docs/api/java/sql/Date.html#valueOf-java.time.LocalDate-